packages/discovery-provider/plugins/notifications/src/utils/format.ts

```diff
@@ -82,6 +83,19 @@ export const formatImageUrl = (profilePictureSizes: string, size: number) => {
   return `${getContentNode()}/content/${profilePictureSizes}/${size}x${size}.jpg`
 }
 
+const PROFILE_PICTURE_PLACEHOLDER_URL =
+  'https://download.audius.co/static-resources/email/imageProfilePicEmpty.png'
+
+export const formatProfilePictureUrl = (user: UserBasicInfo) => {
```

Comment:
I'm surprised we don't already have some kind of utility for this. Is this the first email where we include a user profile picture? Or are _all_ of them broken 😅

Reply:
i had the same question lol.. i looked for the use of a placeholder in the repo but i couldn't find any! def possible they are all broken. i do think not having a profile pic is quite rare since we require it in sign up UI